### ISSUE TITLE
Where are the codes for the core part of Incremental Multi-model Selection ?

### ISSUE THREAD

**Issue by CSer-Tang-hao** [IBM/cdfsl-benchmark#2]
I think this is an interesting and inspiring work, but in your released code, I don't find the code for  "Fine-tuning last-k", "Transductive fine-tuning" and "Transfer from Multiple Pretrained Models" as shown in your paper. I hope you can release these parts as soon. I also want to know the start date of the CVPR2020-VL3-Challenge. Looking forward to your reply!

**Comment by yunhuiguo** [comment]
Hi CSer-Tang-hao, we have included the code for ”Transfer from Multiple Pretrained Models". For "Fine-tuning last-k", you can refer to https://discuss.pytorch.org/t/how-the-pytorch-freeze-network-in-some-layers-only-the-rest-of-the-training/7088/9 and modify the code in finetune.py accordingly. For "Transductive fine-tuning", remove pretrained_model.eval() in finetune.py during testing. CVPR2020-VL3-Challenge has already started. You can refer to https://www.learning-with-limited-labels.com/ for more information.

**Comment by CSer-Tang-hao** [comment]
Thank you for your quick reply! Mentioned in  "General information" that "Only ImageNet based models or meta-learning allowed", I don't know if this "ImageNet" contains CUB, CIFAR100, Caltech256, DTD, which are shown in your benchmark. If not, in other words, can the dataset of the source domain only use mini_ImageNet or other subsets?

**Comment by yunhuiguo** [comment]
> Thank you for your quick reply! Mentioned in "General information" that "Only ImageNet based models or meta-learning allowed", I don't know if this "ImageNet" contains CUB, CIFAR100, Caltech256, DTD, which are shown in your benchmark. If not, in other words, can the dataset of the source domain only use mini_ImageNet or other subsets?

Hi @CSer-Tang-hao , for the challenge, the dataset of the source domain only includes miniImageNet. The inclusion of other domains are used for replicating the results of the paper.

**Comment by CSer-Tang-hao** [comment]
> > Thank you for your quick reply! Mentioned in "General information" that "Only ImageNet based models or meta-learning allowed", I don't know if this "ImageNet" contains CUB, CIFAR100, Caltech256, DTD, which are shown in your benchmark. If not, in other words, can the dataset of the source domain only use mini_ImageNet or other subsets?
> 
> Hi @CSer-Tang-hao , for the challenge, the dataset of the source domain only includes miniImageNet. The inclusion of other domains are used for replicating the results of the paper.

Hi @yunhuiguo, thank you for your answer. I also want to know whether to any requirements for network backbone, only use ResNet10 which shown in your paper, or can use any backbone?

**Comment by yunhuiguo** [comment]
Hi @CSer-Tang-hao, the architecture is not restricted to ResNet10. Any backbone is fine.
